docs/openapi.yml

```diff
@@ -1899,6 +1899,11 @@ components:
           description: regex of paths to be ignored for changes
           type: array
           example: [ "/target", "/.m2", "*/.git/*" ]
+        refPaths:
```

Comment:
Why we need another field for `refPaths`? 
I saw your code that the refpath is under `path.join(projectInfo.location, ".cw-refpaths.json");`, so it would be monitored & reported by filewatcherd, since `pathToMonitor`is the project parent folder.

Reply:
This is for letting filewatcherd know about the external files (outside of the project) to monitor for changes.

Reply:
Makes sense. Thanks for the explanation.